src/components/GamesContainer/GameCard/GameCardInfo/GameCardPlatformIcons/GameCardPlatformIcons.jsx

```diff
@@ -1,3 +1,4 @@
+import PropTypes from 'prop-types';
 import styles from './GameCardPlatformIcons.module.css';
 
 import icon1 from '../../../../../assets/GameCard/platforms/dark/icon-01.svg';
```

Comment:
You could use the `@assets` alias to join the assets folder directly

Reply:
done! thanks